docs/articles/documentation/recipes/using-page-model.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: docs
+title: Using Page Model
+permalink: /documentation/recipes/using-page-model.html
+---
+# Using Page Model
+
+Page Model is a test automation pattern that allows you to create an abstraction of the tested page
```

Comment:
Also you can give a link to the detailed Page Object pattern article http://martinfowler.com/bliki/PageObject.html